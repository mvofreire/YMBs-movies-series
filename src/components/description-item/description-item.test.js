import React from "react";
import ReactDOM from "react-dom";
import { DescriptionItem } from ".";
import { shallow } from "enzyme";

describe("DescriptionItem", () => {
  it("Render with no error", () => {
    const div = document.createElement("div");
    ReactDOM.render(<DescriptionItem title="TITLE" content="CONTENT" />, div);
  });

  it("Wrapper must contain <img /> ", () => {
    const wrapper = shallow(
      <DescriptionItem title="TITLE" content="CONTENT" />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
