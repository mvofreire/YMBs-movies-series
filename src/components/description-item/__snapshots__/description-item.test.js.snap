// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DescriptionItem Wrapper must contain <img />  1`] = `
<div
  style={
    Object {
      "color": "rgba(0,0,0,0.65)",
      "fontSize": 14,
      "lineHeight": "22px",
      "marginBottom": 7,
    }
  }
>
  <p
    style={
      Object {
        "color": "rgba(0,0,0,0.85)",
        "display": "inline-block",
        "marginRight": 8,
      }
    }
  >
    TITLE
    :
  </p>
  CONTENT
</div>
`;
